### DISCUSSION TITLE
Import from CSV a tick box to not auto create field lengths (best guess), set "Target type" as varchar(1024)

### DISCUSSION THREAD

**Discussion by SherlockSoftware** [dbeaver/dbeaver#19324] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
The problem is the guess gets it wrong at times. Some data is lot longer and import fails. Not sure which field(s) failed.

**Describe the solution you'd like**
Allow a test option by making a wide column to browse or even extra scan for widest.. find the cultprit data. 

**Describe alternatives you've considered**
A present have to manually relook at data and trial and error this could be the field.. the data issue.

**Additional context**
From early use of this program it has improved. But last 2 our 4 CSV from Excel, took me good amount time to work it out.

**Comment by Matvey16** [comment, marked as answer]
We have an option to increase the amount of sampe rows that would be checked for maximum column lenght during import

![image](https://user-images.githubusercontent.com/82543000/214798427-2a1fe896-d092-42a1-b491-7f3832c49a0b.png)

**Comment by SherlockSoftware** [comment]
Ok.. will check this out...   shown sampling defaults 100 records..  I could change it to say 12000.. 
Quick look at another file it auto changes to 1200..  got it..

**Comment by E1izabeth** [comment]
@SherlockSoftware so the issue is solved for you?

**Comment by E1izabeth** [comment]
It's been a long time since no update on this issue, so I suppose it's solved.
